src/org/labkey/test/tests/list/ListTest.java

```diff
@@ -533,26 +533,27 @@ public void testNameTrimming()
     public void testLongName()
     {
         String listName = "A_+-:''.¡™£¢∞§¶•ªº–≠œ∑´®†¥¨ˆøπ“‘«æ…¬˚∆˙©√ƒ∂ßΩ≈ç√∫µ≤≥÷‹›ﬁﬂ‡°·‚—±⁄€‹›‡‰Æ«»¢∫√∑∏∂";
-        String fieldWithDefault = TestDataGenerator.randomFieldName("With Default");
+        var fieldWithDefault = FieldInfo.random("With Default", ColumnType.String);
         EditListDefinitionPage listEditPage = _listHelper.beginCreateList(getProjectName(), listName);
         listEditPage.manuallyDefineFieldsWithAutoIncrementingKey("Key");
-        listEditPage.addField(new FieldDefinition(fieldWithDefault, ColumnType.String));
+        listEditPage.addField(fieldWithDefault.getFieldDefinition());
         listEditPage.clickSave();
 
         listEditPage = _listHelper.goToEditDesign(listName);
         var page = listEditPage.getFieldsPanel()
                 .expand()
-                .getField(fieldWithDefault)
+                .getField(fieldWithDefault.getName())
                 .clickAdvancedSettings()
                 .clickDefaultValuesLink();
-        var input = Locator.tagContainingText("td", "With Default").followingSibling("td").descendant("input").findElement(page.getDriver());
+        var input = Locator.tagContainingText("td", fieldWithDefault.getName()).followingSibling("td")
+                .descendant("input").findElement(page.getDriver());
```

Comment:
I think this does want the label.
```suggestion
        var input = Locator.tagContainingText("td", fieldWithDefault.getLabel()).followingSibling("td")
                .descendant("input").findElement(page.getDriver());
```
The failure here is because in 25.7 there is an off-by-one error in the random name generator that sometimes trims the first character from the name part. The failure screenshot shows the field "`안(и@ith Default?`"; note the missing "W".

Reply:
Thank you!
I'd spotted the discrepancy between label and namePart, didn't remember that this was resolved later.
I'm assuming the more-general fix would be painful to backport? If you think it's worth doing, I'm willing to do that as well